src/test/regress/input/external_table.source

```diff
@@ -303,6 +303,13 @@ DROP OWNED BY test_role_issue_12748;
 DROP ROLE test_role_issue_12748;
 DROP PROTOCOL dummy_protocol_issue_12748;
 
+-- Test pg_dump's DROP EXTERNAL TABLE
+-- GitHub Issue #1025: https://github.com/arenadata/gpdb/issues/1025
+CREATE EXTERNAL TABLE issue_1025 (a int) location ('file://@hostname@@abs_srcdir@/data/no/such/place/badt1.tbl' ) format 'csv' (DELIMITER '|');
```

Comment:
Mixed style in the test - a part of the query is in the lower case and the other part is in the upper case. I know that the code right above is written in the similar manner. But the code below is all lower-case.

So, let's do not extend the bad tradition. Better to make it in all-lower-case style.